src/modules/SetListCard/components/PlayHistoryItem/PlayHistoryItem.tsx

```diff
@@ -85,20 +88,31 @@ export const PlayHistoryItem: React.FC<PlayHistoryItemProps> = ({
        * NOTE: the styles for the play history item's `::before` pseudo element is in `styles/globals.css`
        * as Tailwind wouldn't properly apply the styles
        */}
-      <div
-        className={`play-history-item relative flex flex-col gap-1 text-[10px]`}
-      >
-        <p>
-          <span className="font-semibold">{formattedDate}</span>
-          <span> for </span>
-          <span className="font-semibold">{eventType}</span>
-        </p>
-        <p className="flex gap-1 text-slate-500">
-          <span className="">Played in</span>
+      <div className={`play-history-item relative flex flex-col gap-1`}>
+        <div>
+          <Text asElement="span" style="small-semibold">
+            {formattedDate}
+          </Text>
+          <Text asElement="span" style="small">
+            {" "}
+            for{" "}
```

Comment:
Nitpick: I wonder if there's a better way to add the space instead of explicitly hard-coding it here 🤔

Reply:
Added a `FIXME` comment to address in a future PR